LanguageFeatures/Augmentation-libraries/augmenting_types_A06_t07_lib.dart

```diff
@@ -22,6 +22,6 @@ augment library 'augmenting_types_A06_t07.dart';
 augment class C1 extends A {}
 
 augment abstract class C2 extends A {}
-//                                ^
+
+// [error line 28, column 47]
```

Comment:
Is it really true that the error is reported at a location which is beyond the end of the file? If it is actually the observed behavior then it sounds like that would be a thing to fix in the tool, rather than a thing to expect and preserve for an unknown amount of time...

Reply:
Thank you! Seems that this error is obsolete. Line 28 was the last empty line of the file, but this is not relevant any longer